docs/user-guide/semdedup.rst

```diff
@@ -41,14 +41,15 @@ Semantic deduplication in NeMo Curator can be configured using a YAML file. Here
     cache_dir: "semdedup_cache"
     num_files: -1
     id_col_name: "id"
-    id_col_type: "int"
+    id_col_type: "int" # or "str" if the `add_id` module was used
     input_column: "text"
 
     # Embeddings configuration
     embeddings_save_loc: "embeddings"
     embedding_model_name_or_path: "sentence-transformers/all-MiniLM-L6-v2"
     embedding_batch_size: 128
     embedding_max_mem_gb: 25
+    input_file_type: "jsonl" # or "parquet"
```

Comment:
I dont think we should keep here as this is shared among modules. 

I think we should probably also get rid of `input_column`, `id_col_name`, `id_col_type`, what do you think ?